manifests/k/KeeWeb/KeeWeb/1.18.7/KeeWeb.KeeWeb.locale.en-US.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: KeeWeb.KeeWeb
+PackageVersion: 1.18.7
+PackageLocale: en-US
+Publisher: KeeWeb
+PublisherUrl: https://github.com/keeweb/keeweb
+PublisherSupportUrl: https://github.com/keeweb/keeweb/issues
+PrivacyUrl: https://keeweb.info/privacy-policy.html
+Author: KeeWeb
+PackageName: KeeWeb
+PackageUrl: https://keeweb.info
+License: MIT
+LicenseUrl: https://github.com/keeweb/keeweb/blob/master/LICENSE
+Copyright: Copyright (c) 2021 Antelle https://antelle.net
+CopyrightUrl: https://github.com/keeweb/keeweb/blob/master/LICENSE
+ShortDescription: Free cross-platform password manager compatible with KeePass.
+Description: This webapp is a browser and desktop password manager compatible with KeePass databases. It doesnt require any server or additional resources. The app can run either in browser, or as a desktop app.
+Moniker: keeweb
+Tags:
+- electron
+- keepass
+- password
+- password manager
+- keypass
+- foss
+- open source
+- cross-platform
+- javascript
+- security
```

Comment:
```suggestion
- electron
- keepass
- password
- password-manager
- keypass
- foss
- open-source
- cross-platform
- javascript
- security
```

Reply:
Should start to check these before confirming in the Yaml-Script...